### PR TITLE
Cleaned up man page

### PR DESCRIPTION
**Pull request by eNV25**
Some of the changes:

- Removed `% Author` metadata header because there is already `AUTHOR` section in man page.
- Escaped some special characters
- Use `: line` for pandoc's special indenting.
- Always use 4 spaces for code.
- Fix and example where is said `0 = 1` and not `0 = 0`

### CODE REVIEW

**Comment by eNV25** [comment]
According to the pacdoc man page, the `: line` feature is actually called "Definition lists" and come from "PHP Markdown Extra". These just happen to be useful when writing man pages.

**Comment by rvaiya** [comment]
Thanks. Most of the changes are fine, though some of them are gratuitous (e.g using _ instead of *). I have included the subset which I think are appropriate. In the future please create a separate commit if you are going to reformat paragraphs as it adds a lot of noise to the diff.

**Comment by eNV25** [comment]
Some of these changes were from automatically formatted.

Also rejecting some of these changes is fine, but I think it's better if you suggest changes and I make the changes myself. There are some changes I wanted merged but are not there in the final version and it's hard to follow the git diffs. I will need to make a second pull request for that.

Also I will make separate commits to make all changes clear from now on.

**Comment by rvaiya** [comment]
> Also rejecting some of these changes is fine, but I think it's better if you suggest changes and I make the changes myself. 

Fair enough.

> Also I will make separate commits to make all changes clear from now on.

Thanks. It's only necessary if you auto format a large chunk of text in addition to your changes. You should avoid doing this anyway without good reason, but if you do a separate commit makes it easier to distinguish from the other changes.
